test/fuse/linux/fuse_base.h

```diff
@@ -124,6 +127,18 @@ class FuseTest : public ::testing::Test {
   // data from server.
   void GetServerActualRequest(std::vector<struct iovec>& iovecs);
 
+  // Called by the testing thread to query the number of unsent requests in
+  // the requests_ serial buffer of the FUSE server.
+  uint32_t GetServerNumUnsentRequests();
```

Comment:
UnsentRequest or unreceivedRequest

Reply:
Actually the requests are already received by the FUSE server, but they haven't been consumed by the testing thread. How about UnconsumedRequests?